docs/csharp/programming-guide/exceptions/codesnippet/CSharp/using-exceptions_3.cs

```diff
@@ -1,27 +1,32 @@
-        static void TestCatch2()
+        using System;
+        using System.IO;
+
+        public class ExceptionExample
         {
-            System.IO.StreamWriter sw = null;
-            try
+            static void Main()
             {
-                sw = new System.IO.StreamWriter(@"C:\test\test.txt");
-                sw.WriteLine("Hello");
-            }
+                try
+                {
+                    using(var sw = new StreamWriter(@"C:\test\test.txt"))
+                    {
+                       sw.WriteLine("Hello");
+                    }   
+                }
+                // Put the more specific exceptions first.
+                catch (DirectoryNotFoundException ex)
+                {
+                    System.Console.WriteLine(ex.ToString());  
```

Comment:
There's `using System;` at the top, so just `Console.WriteLine` (without `System.`) can be used here.

Also, `.ToString()` is not necessary.